packages/blade/src/components/Alert/_decisions/decisions.md

```diff
@@ -0,0 +1,204 @@
+# Alert
+
+Alerts are messages that communicate information to users about any significant changes or explanations inside the system in a **prominent way**.
+
+This document outlines the API of `Alert` component.
+
+<img src="./alerts-thumbnail.png" width="380" alt="" />
+
+## Design
+
+- [Figma - Alerts](https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?node-id=6824%3A0)
+
+> **Note**
+>
+> The following needs work:
+>
+> 1. color on banking theme in high contrast
+> 2. a11y contrast ratio for notice alerts in high contrast
+> 3. the borderless variant mentioned in full bleed layout shouldn't have any borders and border radii
+
+## API
+
+Annotated component with props:
+
+![Alert breakdown](./alert-breakdown.png)
+
+Sample usage:
+
+```jsx
+import { Alert, Link } from '@razorpay/blade';
+
+<Alert
+  title="International Payments Only"
+  description={
+    <>
+      Currently you can only accept payments in international currencies using PayPal. You cannot
+      accept payments in INR (₹) using PayPal. <Link href="https://razorpay.com">Know More</Link>
+    </>
+  }
+  actions={{
+    primary: { text: 'Primary Action', onClick: {() => { /* do something */ }} },
+    secondary: { text: 'Link', onClick: {() => { /* do something */ }}, href: 'https://razorpay.com'},
+  }}
+/>;
+```
+
+### Alert
+
+We'll expose an `Alert` component with the following API:
+
+| Prop          | Type                                                        | Default     | Description                                                                                | Required |
+| ------------- | ----------------------------------------------------------- | ----------- | ------------------------------------------------------------------------------------------ | -------- |
+| description   | `string`, `JSX`                                             | `undefined` | Body content                                                                               | ✅       |
+| title         | `string`                                                    | `undefined` | A brief heading                                                                            |          |
```

Comment:
Shouldn't title be required by design? Would look odd without one

Reply:
We have use cases for Alerts without a title - [ref](https://github.com/razorpay/blade/blob/div/alerts-decisions/packages/blade/src/components/Alert/_decisions/decisions.md#without-title)